tests/syntax/test_errors.py

```diff
@@ -133,14 +110,14 @@ def test_incomplete_infix_operator():
 
 def test_incomplete_infix_package():
     """Packaged (3+-ary) infix operator with too few arguments."""
```

Comment:
We probably should rename these tests & change the docstrings, since the parser no longer has a concept of argument packages. For example this one could just be `test_incomplete_ternary_operator`.

Reply:
Fixed in c2f474ef3c37f0771756c051029443faec6bc5bb